docs/en/integrations/language-clients/java/index.md

```diff
@@ -170,6 +294,35 @@ try (ClickHouseClient client = ClickHouseClient.newInstance(ClickHouseProtocol.H
 }
 ```
 
+### Features
+#### Compression
+
+The client will by default use LZ4 compression, which requires this dependency:
```

Comment:
If it's relevant for both clients, I see no problem in duplicating the information. especially if we split them into separate pages